templates/tease-search.twig

```diff
@@ -90,6 +90,9 @@
 			</div>
 
 			<p class="search-result-item-content">{{ post.post_excerpt|default(post.post_content)|excerpt( 25 )|raw }}</p>
+			{% if is_archive and fn('is_user_logged_in') %}
+				<a href="{{ fn('get_edit_post_link', post.id)|raw }}">{{ __('Edit this', 'planet4-master-theme') }}</a>
```

Comment:
The "Edit this" link is visible in load more results for non logged-in users as well.
![image](https://user-images.githubusercontent.com/5357471/79869944-cc8e5300-83ff-11ea-9164-7f1b6e834c7d.png)

Reply:
Updated it so that it checks the permissions. Great catch :+1: